compiler-rt/lib/sanitizer_common/sanitizer_platform.h

```diff
@@ -415,9 +415,9 @@
 
 // Enable offline markup symbolizer for Fuchsia.
 #if SANITIZER_FUCHSIA
-#  define SANITIZER_SYMBOLIZER_MARKUP 1
+#  define SANITIZER_SYMBOLIZER_MARKUP_FUCHSIA 1
```

Comment:
Rather than renaming this, could this just remain `SANITIZER_SYMBOLIZER_MARKUP` and anything that's fuchsia-specific below check for `SANITIZER_FUCHSIA` instead?

Reply:
This is always guarding fuchsia specific code or code we don't want to compile for fuchsia, hence the rename. I could get rid of this definition and just use SANITIZER_FUCHSIA if that is more clear.